tensorflow/python/keras/losses.py

```diff
@@ -1415,6 +1415,7 @@ def categorical_hinge(y_true, y_pred):
   return math_ops.maximum(0., neg - pos + 1.)
 
 
+@keras_export('keras.metrics.huber', 'keras.losses.huber')
```

Comment:
Can we only add it to losses namespace for now? Also you will need to update golden files '.pbtxt' files for API changes.

Reply:
Can you please let me know what `.pbtxt` files need changes?